database/migrations/001-punishments-schema.sql

```diff
@@ -0,0 +1,32 @@
+-- Up
+-- The column rest is used to make the punishments database extensible
+
+CREATE TABLE IF NOT EXISTS punishments (
```

Comment:
is the key for both these tables really all fields? also, there should probably be indexes here?

Reply:
> is the key for both these tables really all fields? 

I think so? I don't really know if a user can have two punishments globally or in a room? Maybe @bumbadadabum or @Zarel can weigh in.

>  there should probably be indexes here?

The only time where we query the database we retrieve _all_ the database (to cache them in maps), so I don't really see the use of indexes here.